lib/hotbar.js

```diff
@@ -78,8 +100,8 @@ class HotBar extends LitElement {
       left: 50%;
       height: calc(256px * 4);
       width: calc(256px * 4);
-      transform: translate(calc(-182px * 2), calc(-22px * 4));
-      clip-path: inset(0px calc(74px * 4) calc(234px * 4) 0px);
+      transform: translate(-35.555%, -8.594%);
+      clip-path: inset(0% 28.906% 91.406% 0%);
```

Comment:
Please avoid making _magic numbers_, using `calc()` will improve readability a much.

Reply:
The problem here is that the inset numbers really truly are magic numbers that are specific to the widgets image. I'll see what I can do about adding calc to the transforms.

Reply:
I've made the transforms using `calc()`. The insets are by nature magic numbers so I've made comments explaining how to make them.